src/memory/AmdFlash.hh

```diff
@@ -225,35 +266,35 @@ SERIALIZE_CLASS_VERSION(AmdFlash, 3);
 
 namespace AmdFlashChip
 {
-	using Chip = AmdFlash::Chip;
+	using Chip = AmdFlash::ValidatedChip;
```

Comment:
This confused me. Not this line itself, but the lines below which use `Chip`, which is then actually a `ValidatedChip`. Maybe change this to `using ValidatedChip = ...`?